cytominer_eval/tests/test_evaluate.py

```diff
@@ -202,7 +208,9 @@ def test_evaluate_grit():
     top_result = (
         grit_results_df.sort_values(by="grit", ascending=False)
         .reset_index(drop=True)
-        .iloc[0,]
```

Comment:
Is this black? 
it doesn't seem like a helpful change in code to me. Same goes for the lines below.

Reply:
Yep, it is, just checked it. I do not know why black is only modifying this now...

It is definitely a non-functional change, but it's also harmless. I think we can ignore it